File: backend/app/Models/Subtask.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\Relations\HasMany;

class Subtask extends Model
{
    use HasFactory;

    public $timestamps = false;

    protected $fillable = [
        'task_id',
        'order',
        'question',
        'options',
        'type',
        'marks',
    ];

    public function task() : BelongsTo
    {
        return $this->belongsTo(Task::class);
    }

    public function answers() : HasMany
    {
        return $this->hasMany(Answer::class);
    }
}

File: tests/BolcsessegBarkaAPITests/ApiLibrary/Subtask.cs
using Newtonsoft.Json;

namespace ApiLibrary;

public class Subtask
{
    [JsonProperty("id")]
    public int Id { get; set; }
    
    [JsonProperty("order")]
    public int Order { get; set; }
    
    [JsonProperty("question")]
    public string Question { get; set; }
    
    [JsonProperty("options")]
    public List<string>? Options { get; set; }
    
    [JsonProperty("type")]
    public string Type { get; set; }
    
    [JsonProperty("marks")]
    public double Marks { get; set; }
}